data/shopify_liquid/plus_labels.json

```diff
@@ -0,0 +1,15 @@
+[
```

Comment:
I allowed myself to remove `page_title` from `plus_labels.json` (f.k.a. `plus_objects.yml`), because `page_title` is already present in "general" objects.

Reply:
🚀 Nice!